backlog.txt

```diff
@@ -0,0 +1,49 @@
+ISSUES:
```

Comment:
Thanks for documenting all these issues! I would recommend instead of adding them to a backlog.txt file, let's add them as "Github Issues" here: https://github.com/AustinVaday/SFH-App/issues